### PR TITLE
S2U-42 SAK-49260 Card Game: Keep the photo after a wrong response

### PR DESCRIPTION
**Pull request by stetsche**
This addresses [SAK-49260](https://sakaiproject.atlassian.net/browse/SAK-49260) for [S2U-42](https://sakaiproject.atlassian.net/browse/S2U-42)

[SAK-49260]: https://sakaiproject.atlassian.net/browse/SAK-49260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[S2U-42]: https://sakaiproject.atlassian.net/browse/S2U-42?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by stetsche** [comment]
I found the issue in a missing permission. The code is fine.

File: roster2/bundle/src/bundle/card_game_ca.properties [hunk 1]
```diff
@@ -2,6 +2,7 @@ check_hit_info = Correcte! L\u0027estudiant era {0}!
 check_miss_info = Ups, aix\u00F2 est\u00E0 malament! El nom d\u0027aquest estudiant \u00E9s {0}.
 check = Comprobar
 close = Tancar
+continue = Continua
```
Comment (bgarciaentornos) [inline roster2/bundle/src/bundle/card_game_ca.properties:5]:
```suggestion
continue = Continuar
```
If the previous cases use the infinitive and the Spanish does too I think it's better like this

File: webapi/src/main/java/org/sakaiproject/webapi/controllers/CardGameController.java [hunk 2]
```diff
@@ -75,16 +82,32 @@ public class CardGameController extends AbstractSakaiApiController {
...
         checkSite(siteId);
 
         HashMap<String, Object> config = new HashMap<>();
-        config.put("minAttempts", serverConfigurationService.getInt("cardgame.minAttempts", MIN_ATTEMPTS_DEFAULT));
-        config.put("minHitRatio", serverConfigurationService.getDouble("cardgame.minHitRatio", MIN_HIT_RATIO_DEFAULT));
-        config.put("showOfficialPhoto", serverConfigurationService.getBoolean("cardgame.showOfficialPhoto",
-                SHOW_OFFICIAL_PHOTO_DEFAULT));
+        config.put("minAttempts", minAttempts);
+        config.put("minHitRatio", minHitRatio);
+        config.put("showOfficialPhoto", showOfficialPhoto);
```
Comment (bgarciaentornos) [inline webapi/src/main/java/org/sakaiproject/webapi/controllers/CardGameController.java:111]: Not sure where this is called but should skipNoImageUser be added here too?

File: roster2/bundle/src/bundle/card_game_ca.properties (hunk 1 above)
Comment (stetsche) [reply roster2/bundle/src/bundle/card_game_ca.properties:5]: I copied this translation from another bundle, but I'll commit your change. Thanks

File: webapi/src/main/java/org/sakaiproject/webapi/controllers/CardGameController.java (hunk 2 above)
Comment (stetsche) [reply webapi/src/main/java/org/sakaiproject/webapi/controllers/CardGameController.java:111]: No, it's not needed. This map defines the config the frontend needs. The frontend does not need to know about skipNoImageUser for now, the users are filtered on the backend.

**Review by bgarciaentornos — APPROVED** [review]

**Review by mpellicer — APPROVED** [review]
Thanks for the review @bgarciaentornos.

**Pull request merged**